app/models/user.rb

```diff
@@ -153,13 +153,11 @@ class User < ApplicationRecord
     :database_authenticatable,
     :invitable,
     :lockable,
-    :omniauthable,
     :recoverable,
     :rememberable,
     :timeoutable,
     :trackable,
-    :validatable,
-    omniauth_providers: [:github]
+    :validatable
```

Comment:
I believe we should have a trailing comma here

Reply:
`./bin/standardize` yells at us for this.